src/components/App/AppHead.tsx

```diff
@@ -274,7 +283,7 @@ export const AppHead = (props: IProps) => {
             <LinearProgress id="prog" variant="determinate" value={complete} />
           </div>
         )}
-        {(!busy && !doSave) || complete !== 0 || (
```

Comment:
This is the only line that concerns me. The reason we checked for complete != 0 was to prevent both the determinant and indeterminant bars from showing at the same time. I think removing this will show both when the linear progress is happening.